### DISCUSSION TITLE
Schema change for enum properties after update.

### DISCUSSION THREAD

**Discussion by Arcus16** [mikro-orm/mikro-orm#4964] [General]
Hi, I updated the MikroOrm version in my application, and to my surprise, it generated a new migration after the update, and all changes look like:
```typescript 
    this.addSql('alter table `some_table` modify `column1` enum(\'value1\', \'value2\') not null, modify `column2` enum(\'value3\', \'value4\') not null;');
 ...
```
The change in the snapshot is about removing white space between enum values. So, after a little digging, I found out that this change was introduced in version 5.8.0, and I think it was in [this commit](https://github.com/mikro-orm/mikro-orm/pull/4601/files#diff-4bb9c37dbbc7b6fcc105c9d554d2824c9a528ab6067a7ab9a890634e03534688L208).

I think it's okay to just run this migration and only commit the change in the snapshot since "nothing changed" and I don't want to run this migration (that does in the end, nothing) on a table with millions of records.

So my question is, is there some functional reason to remove the space?

**Comment by B4nan** [comment]
Just wipe your snapshot if it's caused by that, it will be regenerated when you create a new migration again.
